tests/integration/kubernetes/k8s-nvidia-cuda.bats

```diff
@@ -62,5 +62,13 @@ teardown() {
     echo "=== CUDA vectoradd Pod Logs ==="
     kubectl logs "${POD_NAME_CUDA}" || true
 
-    teardown_common "${node}" "${node_start_time:-}"
+    kubectl describe pods
+
+    # Clean up resources
+    [ -f "${pod_yaml}" ] && kubectl delete -f "${pod_yaml}" --ignore-not-found=true
+
+    if [[ -n "${node_start_time:-}" && -z "${BATS_TEST_COMPLETED:-}" ]]; then
+        echo "DEBUG: system logs of node '${node}' since test start time (${node_start_time})"
+        exec_host "${node}" journalctl -x -t "kata" --since '"'"${node_start_time}"'"' || true
+    fi
```

Comment:
Rather than copy-pasting this code, did you consider adding a new function e.g. `print_node_journal_since_test_start` in test_common.sh?